src/parser.js

```diff
@@ -643,11 +642,23 @@ function mapStatement(node, meta) {
   return b.expressionStatement(mapExpression(node, meta));
 }
 
+function mapClassPrivateVariables(classNode, meta) {
+  const privateAssignments = classNode.body.expressions.filter(classExpression =>
+    classExpression.constructor.name === 'Assign' && get(classExpression, 'variable.this') !== true
+  ).map(classExpression =>
+    b.expressionStatement(mapExpression(classExpression, meta))
+  );
+
+  return privateAssignments;
```

Comment:
This is usually a sign that I've spent a lot of time debugging the method :)  It doesn't fit with `decaf` style, so off it goes!